app/Tickets/Flipside/class.FlipsideTicketRequestEmail.php

```diff
@@ -41,7 +41,7 @@ public function getHTMLBody()
                 Reminder:  The dates for mailing your ticket request envelope are '.$this->settings['mail_start_date'].' through '
                 .$this->settings['request_stop_date'].'. Your postmark must be within that date range.<br><br>
                 If you need to update your ticket registration before mailing in your ticket request, return to 
-                <a href="https://secure.burningflipside.com/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
+                <a href="'.$this->secureUrl.'/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
```

Comment:
I really don't like the Email and PDF base classes having any concept of the website or it's structure. I have part of a change to fix all that. Will push later this evening.